### ISSUE TITLE
Change the name of "Matrix" to something more unique

### ISSUE THREAD

**Issue by andrew712-1** [matrix-org/matrix-spec#3]
### It's hard to ask someone, do you have a "matrix" account? 

They're like "What do you mean? The movie?" 

Matrix is a term that is used for a lot of things, and it gets awkward and confusing when asking someone such a question.

> ma·trix  (mā′trĭks)
> n. pl. ma·tri·ces (mā′trĭ-sēz′, măt′rĭ-) or ma·trix·es
> 1. A situation or surrounding substance within which something else originates, develops, or is contained: "Freedom of expression is the matrix, the indispensable condition, of nearly every form of freedom" (Benjamin N. Cardozo).
> 2. The womb.
> 3. Anatomy
> a. The formative cells or tissue of a specialized structure such as a hair, nail, claw, or tooth.
> b. See[ ground substance](https://www.thefreedictionary.com/ground+substance).
> 4. Geology
> a. The solid matter in which a fossil or crystal is embedded.
> b. Groundmass.
> 5. A mold or die.
> 6. The principal metal in an alloy, as the iron in steel.
> 7. A binding substance, as cement in concrete.
> 8.
> a. Mathematics A rectangular array of numeric or algebraic quantities subject to mathematical operations.
> b. Something resembling such an array, as in the regular formation of elements into columns and rows.
> 9. Computers The network of intersections between input and output leads in a computer, functioning as an encoder or a decoder.
> 10. Printing
> a. A mold used in stereotyping and designed to receive positive impressions of type or illustrations from which metal plates can be cast. Also called mat2.
> b. A metal plate used for casting typefaces.
> 11. An electroplated impression of a phonograph record used to make duplicate records.

We often compare decentralized services (such as matrix) to email, but the term "email" is much clever and unique. People either know what you are talking about (email) or they just get curious to know what that is while others can understand that it's made of **e + mail** so it must be some kind of **mail**.

Even "XMPP" is a little better than "Matrix" at least because it's unique.

**TL;DR:** Please change the name of "Matrix" to something more unique. How should we ask someone if they have a matrix account? I can't ask them if they have an Element acc, because Element is just a client.

I hope my suggestion will be heard. Thank you.

**Comment by ShadowJonathan** [comment]
> How should we ask someone if they have a matrix account?

"Hey, do you have a matrix account?"

While I agree that, right now, the word "Matrix" has a heavier association to something else than Matrix in both the movie space (for now) and the mathematics space, I don't think it's a huge problem, as natural language was made to work with these kinds of name collisions, by differentiating them in context.

A majority of the definitions for "matrix" you cite are job or expertise-specific, [the original Wiktionary page](https://en.wiktionary.org/wiki/matrix#Noun) also lists a bunch of qualifiers to most of them; Biology, Mathematics, Computing, Electronics, Geology, etc.

These do not necessarily collide with the kind of word that "Matrix" wants to be, which sits more in the same space as "email", but "email" has also become largely erased, now its just "mail", like old postal mail was.

In the end, Matrix also has a bit of a unique situation, while it is more the *fabric* of whatever apps want to interconnect, instead of the front-and-center brand. In that aspect, it is a unique enough name, with a unique enough context, to differentiate itself between all those other usages. (What I mean; Element might become a popular "brand", and so might Fluffychat, but people might care less, or might associate a very specific context, to the word "Matrix" together with that, which is unambiguous to those other usages)

One specific thing where "Matrix" fails on is being unique to catch on, it is fairly close to relevant subjects - computing - but then it doesn't have much leverage after that.

But I don't see that as a flaw, Matrix (as a concept) was not designed (and tbh, shouldnt be) to be instantly perceivable like a brand, like Discord, like Telegram, like Slack. Matrix should be on the same level as email in terms of perceivability.

In that aspect, email has a better head-start, as people already know about mail, matrix doesnt, **but that's fine**, that is not what matters, its more an unique-enough identifier for people in the "back-office" to work with, instead of people "at the store" to recognize, that goes to Element, Fluffychat, Cinny, Beeper, and many more.

The ultimate promise of matrix is to make *all* Instant Messaging apps work together, once that's a given, then it doesn't matter what really is powering it, then all the focus goes to the apps that're loud and bright and aimed to catch the user's eye. The problem with matrix's "perceivability" will only matter as long as matrix is still "coming-up".

But in my opinion, it shouldn't be matrix's "brand" that "talks" about itself, it should be the results, the invisible network effect, the carrying forces of all those apps that seamlessly work together, and people in the back going "i want that, i'm in, its called 'matrix'? whatever, i'm signing up".

**Comment by richvdh** [comment]
Yup. Sorry, no plans to rename in the forseeable future.

> They're like "What do you mean? The movie?"

whatever the protocol is called, you'll get a similar reaction (minus the movie part) from anyone who hasn't yet heard of the protocol.
